monkeys/fqdn-s3/README.md

```diff
@@ -0,0 +1,18 @@
+# Chaos Test: fqdn-s3
+
+![](https://github.com/cilium/chaos-monkeys/raw/master/monkeys/fqdn-s3/.img/fqdn-s3.jpg)
+
+## Test Description
+
+This test make a HTTP HEAD call to a S3 file with a FQDN policy.
+
+The reason of this test is to make sure that a new identity is always added due
+DNS reply changes.
```

Comment:
This might be more descriptive with: The reason for this test is to verify that a new IP is added to the allowed list quickly enough to allow a HTTP connection.
A note explaining that this is testing one thing, but that a failure could happen for unrelated reasons would be good. In particular, if this fails because regeneration is too slow, how to verify that would be helpful (I'm not sure how, actually, beyond pointing the user to the dump of the cilium policy or the endpoint's CEP data).

Separately, could you add a note explaining that this monkey will generate many identities and that might break a cilium cluster. Also mention that `--tofqdns-min-ttl` can be lowered to expire these identities sooner, helping to stabilise the cluster.